core/src/main/java/com/threerings/getdown/data/Application.java

```diff
@@ -587,7 +587,7 @@ else if ((cfgfile = getLocalPath(CONFIG_FILE + "_old")).exists()) {
         }
 
         // check if we're overriding the domain in the appbase
-        _appbase = SysProps.overrideAppbase(_appbase);
+        _appbase = processArg(SysProps.overrideAppbase(_appbase));
```

Comment:
@samskivert alarm! this line breaks versioned app upgrade!

Reply:
``processArg`` replaces ``%VERSION%`` making
```
protected URL createVAppBase (long version)
        throws MalformedURLException
    {
        String url = version < 0 ? _appbase : _appbase.replace("%VERSION%", String.valueOf(version));
        return HostWhitelist.verify(new URL(url));
    }
```
always return current version as ``_appbase`` has already replaced version with the current!

Reply:
@neuweiler did you test versioned app upgrade w/o patches?

Reply:
all our clients with latest getdown are now staring at ``m.unable_to_repair`` error. we should fix this asap

Reply:
created #226

Reply:
@sergiorussia Sorry!!! (see comment on #266)